src/test/java/seedu/storemando/logic/commands/ReminderCommandTest.java

```diff
@@ -60,10 +61,13 @@ public void equals() {
     @Test
     public void execute_multipleItemsFound() {
         ItemExpiringPredicate predicate = new ItemExpiringPredicate((long) 100);
+        String expectedMessage = String.format(ReminderCommand.MESSAGE_SUCCESS_EXPIRING_ITEM, 100, "days");
+        ReminderCommand reminderCommand = new ReminderCommand(predicate, 100L, "days");
         expectedModel.updateFilteredItemList(predicate);
         ItemComparatorByExpiryDate comparator = new ItemComparatorByExpiryDate();
         expectedModel.updateSortedItemList(comparator);
         expectedModel.setItems(expectedModel.getSortedItemList());
+        assertCommandSuccess(reminderCommand, model, expectedMessage, expectedModel);
```

Comment:
Just a suggestion, maybe you can include comments on what the the test is testing more specifically.